turbo.json

```diff
@@ -2,9 +2,44 @@
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build": {
+      "dependsOn": ["^build", "^typecheck"],
+      "inputs": ["src/**/*.ts"],
+      "outputs": ["dist/**"],
+      "outputMode": "errors-only"
+    },
+    "lint": {
+      "inputs": ["src/**/*.ts"],
+      "outputMode": "errors-only"
+    },
```

Comment:
We do not have any lint script in any app or package yet, do we plan to add some?

Reply:
> ⚠️ There is currently no linter configured, despite it being called by the code-review task (it won't fail, it just doesn't run it). We will configure https://github.com/pagopa/dx/pull/4 when it will be merged.

https://github.com/pagopa/dx/pull/4 is currently in review

Reply:
Sorry I missed that part of the description 👍🏽